static/app/views/performance/http/httpDomainSummaryPage.tsx

```diff
@@ -157,6 +169,7 @@ export function HTTPDomainSummaryPage() {
             ]}
           />
           <Layout.Title>
+            {project && <ProjectAvatar project={project} size={36} />}
```

Comment:
<img width="338" alt="image" src="https://github.com/getsentry/sentry/assets/18689448/c2d634f5-adcb-4d14-83db-f23f4e920ebb">

I think this needs `align-items: center`, the avatar is mis-aligned with the name